tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -143,6 +143,14 @@ def kill_all(self):
         self.marionette.switch_to_frame()
         self.marionette.execute_async_script("GaiaApps.killAll();")
 
+    def install_app(self, nameApp, urlApp):
+        self.marionette.switch_to_frame()
+        self.marionette.execute_script(
+            'navigator.mozApps.install("%s")' % urlApp)
+        from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+        confirm_install = ConfirmInstall(self.marionette)
+        confirm_install.tap_confirm()
```

Comment:
there is something data_layer.set_bool_pref('dom.mozApps.auto_confirm_install' @mwargers ?? if it is where could I find it